test/com/amazon/ionschema/ISLforISLTestRunner.kt

```diff
@@ -38,6 +38,8 @@ class ISLforISLTestRunner(
         .withAuthority(AuthorityFilesystem("ion-schema-schemas"))
         .build()
 
+    // There are certain conditions that make a schema invalid that cannot be detected by the ISL for ISL (for example, a schema importing itself).
+    // Test files with "invalid_schema" definitions should be listed here if and only if they are invalid for some reason that cannot be validated by ISL.
```

Comment:
Nit: these comments are wide enough that I had to scroll horizontally in GitHub to read them in full. Consider restacking them?

```suggestion
    // There are certain conditions that make a schema invalid that cannot
    // be detected by the ISL for ISL (for example, a schema importing itself).
    // Test files with "invalid_schema" definitions should be listed here if and
    // only if they are invalid for some reason that cannot be validated by ISL.
```